core/asm/compiler.go

```diff
@@ -162,7 +162,7 @@ func (c *Compiler) compileNumber(element token) (int, error) {
 }
 
 // compileElement compiles the element (push & label or both)
-// to a binary representation and may error if incorrect statements
```

Comment:
This was fine really, "to error" is a verb :)

Reply:
Thank @karalabe . Will work on that.